homework-classes/RepoView.js

```diff
@@ -18,9 +18,40 @@
      * Renders the repository details.
      * @param {Object} repo A repository object.
      */
+
     render(repo) {
-      // TODO: replace this comment and the console.log with your own code
       console.log('RepoView', repo);
+      const repoContainer = document.querySelector('.repo-container');
+      const select = document.querySelector('.repo-select');
```

Comment:
Do not get references like this from the DOM. The container is already available as `this.container`. And you should not reference the `select` element here. That is the responsibility of the `HeaderView` class.